src/main/java/step1/StringAddCalculator.java

```diff
@@ -0,0 +1,18 @@
+package step1;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    public StringAddCalculator(){
+
+    }
```

Comment:
이 기본생성자는 굳이 필요가 없을 것 같네요.

---

static 기능만 제공하는 클래스이니 생성자 접근제어자를 private로 바꾸고 객체생성을 막아버리는 방법으로는 쓸수 있습니다. 😄 

```suggestion
    private StringAddCalculator(){
    }
```